src/Transports.AspNetCore/WebSockets/IOperationMessageProcessor.cs

```diff
@@ -0,0 +1,21 @@
+using GraphQL.Transport;
+
+namespace GraphQL.Server.Transports.AspNetCore.WebSockets;
+
+/// <summary>
+/// Processes a stream of messages received from a WebSockets client.
+/// <see cref="IDisposable.Dispose"/> must be called when the connection terminates.
+/// Methods defined within this interface need not be thread-safe.
+/// </summary>
+public interface IOperationMessageProcessor : IDisposable
```

Comment:
Will `IAsyncDisposable` be useful here?

Reply:
Not at this time.  We could add the interface anyway, but the only thing that happens during disposal is signaling cancellation tokens and disconnecting active subscriptions (meaning, signaling the event source that no more notifications should be provided).

Reply:
OK to keep as is.